galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager.ivt/src/main/java/dev/galasa/zos/manager/ivt/ZosManagerFileDatasetIVT.java

```diff
@@ -124,15 +137,15 @@ private boolean checkContentWasWritten(String pds, String member, String... cont
     //Test that an existing PDS exists
     @Test
     public void testExistingDS() throws ZosBatchException, TestBundleResourceException, IOException, ZosDatasetException {
-    	String desiredDataSetName = "CTS.CICSCOG.JCL";
+    	String desiredDataSetName = "CTS.CICSCOG.JCL" + "." + getRandomString();
     	assertThat(checkThatPDSExists(desiredDataSetName)).isTrue();
     	assertThat(fileHandler.newDataset(desiredDataSetName, imagePrimary).exists()).isTrue();
     }
    
     //Test that a non-existant PDS doesn't exist 
     @Test
     public void testNonExistingDS() throws ZosBatchException, TestBundleResourceException, IOException, ZosDatasetException {
-    	String desiredDataSetName = "CTS.CICSCOG.JCJ";
+    	String desiredDataSetName = "CTS.CICSCOG.JCJ" + "." + getRandomString();
```

Comment:
In this case we are testing an existing dataset that doesn't already exist - we are not creating it so don't need to add the random string

Reply:
I think getting the random string only adds to the "do not conflict with other tests" ethos...? Happy to remove if truly unnecessary - otherwise I think it's a small overhead for a  lot of peace-of-mind?